tools/build_defs/framework.bzl

```diff
@@ -272,12 +273,11 @@ def cc_external_rule_impl(ctx, attrs):
         bin_dir_name = attrs.out_bin_dir,
         lib_dir_name = attrs.out_lib_dir,
         include_dir_name = attrs.out_include_dir,
+        declared_outputs = outputs.declared_outputs,
     )
     return [
         DefaultInfo(files = depset(direct = outputs.declared_outputs)),
         OutputGroupInfo(
-            gen_dir = depset([outputs.installdir]),
```

Comment:
Let's keep gen_dir (further we make it refer to the copy of the directory), and out_binary_files (later each binary will be in its own group)
* Please read the comment to the PR first, with more detailed explanation